test/service/test_manifest.py

```diff
@@ -189,6 +189,7 @@ def test_manifest(self):
             ('donor_organism.provenance.document_id', '', '89b50434-f831-4e15-a8c0-0d57e6baa94c'),
             ('donor_organism.genus_species', '', 'Mus musculus'),
             ('donor_organism.diseases', '', 'subcutaneous melanoma'),
+            # TODO: Revert to `organism_age` https://github.com/DataBiosphere/azul/issues/1907
```

Comment:
This TODO seems out of place, nothing was changed (here) to revert

Reply:
Ah, makes sense. The reality is that in the index, this field `organism_age ` in the test is actually `organism_age_value` and so there is a logical mismatch. I added the TODO as a note to ensure all the locations where the index and the response should be in sync.

Reply:
ok.